config/commands.php

```diff
@@ -56,16 +56,20 @@
     |
      */
 
-    'hidden' => [
+    'hidden' => array_merge([
         NunoMaduro\LaravelConsoleSummary\SummaryCommand::class,
         Symfony\Component\Console\Command\DumpCompletionCommand::class,
-        Symfony\Component\Console\Command\HelpCommand::class,
         Illuminate\Console\Scheduling\ScheduleRunCommand::class,
         Illuminate\Console\Scheduling\ScheduleListCommand::class,
         Illuminate\Console\Scheduling\ScheduleFinishCommand::class,
         Illuminate\Foundation\Console\VendorPublishCommand::class,
         LaravelZero\Framework\Commands\StubPublishCommand::class,
-    ],
+    ], Phar::running() ? [
```

Comment:
ℹ️ Neat little trick to hide/remove some commands in the context of the PHAR only